bundles/org.openhab.binding.http/src/main/java/org/openhab/binding/http/internal/http/RateLimitedHttpClient.java

```diff
@@ -29,16 +47,24 @@
  */
 @NonNullByDefault
 public class RateLimitedHttpClient {
+    private final Logger logger = LoggerFactory.getLogger(RateLimitedHttpClient.class);
     private static final int MAX_QUEUE_SIZE = 1000; // maximum queue size
-    private HttpClient httpClient;
-    private int delay = 0; // in ms
+    private Thing thing;
```

Comment:
Please keep client and thing separated. Why did you remove the client from the constructor?

Reply:
The intention was to keep the previous log entries and they got the Thing-id. 
Should i use a diffrent log entry or pass the thing id?